ibis/tests/all/test_window.py

```diff
@@ -132,6 +132,11 @@
             lambda gb: gb.float_col.cummax(),
             id='max',
         ),
+        param(
+            lambda t, win: t.double_col.count().over(win),
+            lambda gb: gb.double_col.cumcount(),
+            id='count',
```

Comment:
It looks like this test is failing in CI. You can mark it as `pytest.mark.xfail_backends([Csv, Impala, Pandas, Parquet, PostgreSQL])` for now and I will take a look at it after we merge this.

Reply:
sounds good. thanks!